index.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+require_once 'functions.php';
+
+$db = new PDO('mysql:host=db; dbname=lily-collection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->prepare("SELECT `item-name`,`image-URL`,`number-of-pieces`,`age-category`,`star-rating`,`buy-URL`,`retired` FROM `lego-sets`");
```

Comment:
preparing statement/execute/fetching all could be one function eg: retrieveLegoSets()